README.md

```diff
@@ -89,6 +99,9 @@ git commit -a                           # Make a new commit and automatically "a
 git commit -am "I still do!"            # A combination of the above
 git commit --amend                      # Re-do the commit message of the previous commit (don't do this after pushing!)
                                         #   We _never_ change "public history"
+git reset <file>                        # Unstage a staged file leaving in working directory without losing any changes.
+git reset --soft [commit_hash]          # resets the current branch to <commit>. Does not touch the staging area or the working tree at all. 
+                                        # --hard mode would discard all changes to given file.
```

Comment:
This is still a confusing and technically incorrect description. The " to given file." does not make sense for a `git reset --hard [commit_hash]` statement, as a reset hard throws away all changes in all tracked files.

I am sorry to be so pedantic about this, we we don't want the cheat sheet to cause more confusion than necessary.